src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBarracksTower.java

```diff
@@ -11,51 +11,60 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Building class of the BarracksTower.
+ * Building class for the Barracks Tower.
  */
+@SuppressWarnings("squid:MaximumInheritanceDepth")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:MaximumInheritanceDepth' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)